components/src/atoms/breadcrumbs/Breadcrumbs.html

```diff
@@ -0,0 +1,35 @@
+<div class="breadcrumbs">
+  <template v-for="(breadcrumb, i) in breadcrumbs">
+    <template v-if="breadcrumbs.length === i + 1">
+      <!-- @slot Current page (Scoped slot) -->
+      <slot
+        name="current"
+        :breadcrumb="breadcrumb"
+      >
+        <span
+          :key="breadcrumb.text"
```

Comment:
Text of breadcrums will not always be unique, e.g. we can have the same name for category and product. It's the same case as here https://github.com/SnowdogApps/alpaca-ui/pull/161#discussion_r276017228. We should add id to breadcrumbs mock or use index with some prefix `:key="breadcrumb-${i}"`

Reply:
I will use `:key="breadcrumb-${i}"` for now ;)